### PR TITLE
Use theme colors.

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

Waterfall PR start, to gradually further improve the CLAIM flow styling. This one:
- Refactors the claim styles, by using the CowSwap theme colors instead.
- Show `0 CoW` by using the 0 as a fallback number.
Screen Shot 2022-01-12 at 15 54 36
Screen Shot 2022-01-12 at 15 54 30

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @biocom , great changes!

Some questions:
1. Claim page itself looks fine, but when compare it with the 'Swap' page, espesially in a mobile view, it becomes highly visible that we use different boarder radius for forms on pages
claim

swap

2. As for the background, we also use mix of designs for Swap/Content pages on the Claim page. I mean, there is a clear background+black boarder on the form on the Swap page; blurred background and almost transaparen section boarder on a content page. But on the Claim page we use blurred background and the Black boarder. Should we leave it as it is? 
3. I'm not sure if this issue is related to the current PR or not, but it would be nice to remove 'token' from the validation message
image

Thanks!

**Comment by elena-zh** [comment]
4. Grid lines are visible in the dark mode and not visible in the light mode. Could we make them visible on both themes? 
image
image

5. The same with instructions on the Approve tokens page. 
image
image
But I'd remove a different color there

6. I would change checkboxes color to match more or less the themes colors

7. Maybe we could make the positive message look brighter a bit? Looks a bit depressive to me..
image

**Comment by elena-zh** [comment]
I did not report issue related to a mobile view, due to as far as I can understand, it will be covered in another PR.

File: src/custom/pages/Claim/ClaimSummary.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default function ClaimSummary({ hasClaims, unclaimedAmount }: ClaimSummar
         <div>
           <ClaimTotal>
             <b>Total available to claim</b>
-            <p>{formatSmart(unclaimedAmount)} vCOW</p>
+            <p>{formatSmart(unclaimedAmount) || 0} vCOW</p>
```
Comment (alfetopito) [inline src/custom/pages/Claim/ClaimSummary.tsx:34]: Thanks for that 👍

**Review by alfetopito — APPROVED** [review]
Looking absolutely gorgeous 10/10 would claim again

**Review by anxolin — APPROVED** [review]
nice!

**Comment by W3stside** [comment]
cool! seeing border radius issues, maybe weren't fixed yet:
image

**Comment by biocom** [comment]
> cool! seeing border radius issues, maybe weren't fixed yet: image

Yes one of the things I still need to polish...

**Comment by elena-zh** [comment]
I noticed that in the current PR something wrong with claim amounts: 
1. the app still shows available to claim tokens when they have beed already claimed


2. Airdrop claim option is not displayed in the investment grid
missing

**Pull request merged**